docs/current_docs/quickstart/conclusion.mdx

```diff
@@ -10,6 +10,8 @@ title: "Conclusion"
 
 This quickstart has walked you through the process of Daggerizing an example application. It has taught you key Dagger concepts and introduced you to the Dagger API, the Dagger CLI, Dagger Functions and the Daggerverse.
 
+Lastly, you should now have a [Dagger Cloud](https://dagger.cloud/) account, which helps you visualize your pipelines and gives a unique view of all pipelines, both pre-push and post-push. This will help as you continue on your Dagger journey.
```

Comment:
> which helps you visualize your pipelines and gives a unique view of all pipelines

this reads a bit odd to me as it has the `pipelines` repetition, does it need rewording?